portal-web/test/functional/com/liferay/portalweb/macros/Form.macro

```diff
@@ -311,6 +311,74 @@ definition {
 			value1 = "${successPageDescription}");
 	}
 
+	macro editTextAdvancedTab {
```

Comment:
Good job, @ewertonluna! :slightly_smiling_face: 

Have you created this macro because other tests will need to interact with the fields existing in the Advanced Tab? If this is not the case, can you please just add to this macro the code demanded by your test?

In addition, please use the same terms available on the Forms Sidebar, for example, instead of use `Predefined` in `key_fieldPredefinedText` let's use `PredefinedValue` because this is the term used on the Forms Sidebar. Can you please visit this macro with this tip in mind?

Thanks.

Reply:
Hi, @natocesarrego! I created this macro for future use in case someone needs to interact with more than just one field in the Advanced Tab for the Text Field, so they will be able to interact with any field they want at once.
If you want me to remove it just let me know! :)

Regarding the correct term for variables, just to make sure I understand, instead of `key_fieldPredefinedText` it should be nammed `key_fieldPredefinedValueText`, right? Or just key_fieldPredefinedValue?

Thank you!

Reply:
Thanks for your clarifications, @ewertonluna. :slightly_smiling_face: 

For this pull, let's add only what it demands. :wink: 

Always try to use the same term / word used on the UI, so  `key_fieldPredefinedValue` looks great!

Thanks.